app/models/internal/impl/DefaultFeatures.java

```diff
@@ -69,8 +72,23 @@ public boolean isReportsEnabled() {
     }
 
     @Override
-    public ImmutableList<String> getSourceTypes() {
-        return _sourceTypes.stream().map(SourceType::name).collect(ImmutableList.toImmutableList());
+    public ImmutableList<SourceType> getReportingSourceTypes() {
```

Comment:
It seems odd that all of these methods return concrete types but I've seen this elsewhere in the repo, specifically with `ImmutableList` so 🤷‍♂

Reply:
Why is this odd that they return types (an enum) instead of strings? I would have thought returning types was the norm and returning strings was strange. Can you elaborate?

Reply:
I meant that it returns `ImmutableList` vs `List`

Reply:
That's fair. This is the convention we adopted for the ArpNet/ISM projects since imo it was clearer what the intention was and we wanted to be clear on where data was mutable.